app.css

```diff
@@ -1,3 +1,3 @@
 (max-width:1024px){.personal-details__form-table{display:block}.personal-details__tr-row{display:block}.personal-details__td-cell{display:block}.personal-details__td-label{min-width:200px;display:block;padding-bottom:24px}.personal-details__td-value{display:block;width:100%}.personal-details__td-cell+.personal-details__td-cell{padding-left:0}.personal-details .age-property{margin-bottom:20px}.personal-details .age-property .personal-details__td-label{margin-bottom:20px}}.avatar-and-interests{font-size:14px;font-size:.875rem}.avatar-and-interests .page-indicator{margin-bottom:0;margin-top:0;position:relative;top:-16px}.avatar-and-interests__form{background-color:#fff;border-radius:8px;padding:32px 32px 40px 40px;width:438px}.avatar-and-interests .checkbox{margin-right:8px}.avatar-and-interests__section+.avatar-and-interests__section{border-top:1px solid #ecf0f1;margin-top:32px;padding-top:32px}.avatar-and-interests__title{font-size:18px;font-size:1.125rem;font-weight:400;margin:0 0 24px 0}.avatar-and-interests__text{margin:0}.avatar-and-interests__avatar-box{height:130px;width:130px;background-color:#dbdbdb;border-radius:8px;background-image:url(images/avatar_placeholder.png);background-size:cover}.avatar-and-interests__avatar-image{width:100%;height:100%;background-size:cover;background-position:50% 50%;border-radius:8px}.avatar-and-interests__avatar-file{display:none}.avatar-and-interests__spinner{position:absolute;left:50%;top:50%}.avatar-and-interests__upload-button{color:#7f8c8d}.avatar-and-interests__interests-list{-moz-column-count:4;-moz-column-gap:20px;-webkit-column-count:4;-webkit-column-gap:20px;column-count:3;column-gap:20px}.avatar-and-interests__interests-list__item{margin-bottom:8px}.avatar-and-interests__avatar-upload-cell{vertical-align:top;position:relative}.avatar-and-interests__avatar-upload-button{font-size:14px;font-size:.875rem;vertical-align:bottom;height:32px;line-height:32px;position:absolute;bottom:0}.avatar-and-interests__errors-list{margin:0;padding:0;margin-top:8px}.avatar-and-interests__error{color:#29c566}.avatar-and-interests-page{text-align:center}.avatar-and-interests-page .avatar-and-interests{text-align:left;display:inline-block;display:-webkit-inline-box}.avatar-and-interests-page__buttons-row{text-align:center;margin-top:8px}.avatar-and-interests-page__buttons{width:438px;display:inline-block}.avatar-and-interests-page__buttons .button{display:inline-block}.slider{position:relative}.slider__track{background-color:#0c58da;border-radius:2px;height:4px;left:0;position:absolute;right:0;top:50%;width:100%}.slider__handle{background-color:#0c58da;border-radius:50%;color:#fff;cursor:default;height:48px;left:50%;line-height:48px;margin-left:-24px;margin-top:-22px;position:absolute;text-align:center;top:50%;user-select:none;width:48px}.spinner{position:relative}.spinner__element{width:9px;height:3px;background-color:#000;border-radius:3px;box-sizing:border-box;position:absolute;top:0;left:0;transform-origin:0 0;anim
```

Comment:
Change should be made in the custom_styles.css file

Reply:
made the requested changes https://github.com/SeshanPillay25/fixing-userinyerface/pull/17
@SeshanPillay25